scanapp/models.py

```diff
@@ -31,172 +32,125 @@
 from rest_framework.authtoken.models import Token
 
 
-# Create your models here.
 @receiver(post_save, sender=settings.AUTH_USER_MODEL)
 def create_auth_token(sender, instance=None, created=False, **kwargs):
     if created:
         Token.objects.create(user=instance)
 
 
-class ScanInfo(models.Model):
+class Scan(models.Model):
+    """
+    Store various attributes of a scan
+    """
     def __str__(self):
-        return self.scan_type
+        return self.scan_directory
```

Comment:
I am not sure this is a good representation for a Scan. The URL would be better IMHO. And for an upload as opposed to a URL, then the URL could be a `file:///xxxx` URL